GoMeet/src/main/webapp/js/create-meeting-step1.js

```diff
@@ -0,0 +1,41 @@
+let guestsAdded = 0; // Number of guest email inputs on the page 
+
+/**
+ * Add an input element to the guest list for user to add email 
+ * @param {*} doc the document object to be manipulated 
+ */
+function addGuest(doc) {
+  guestsAdded += 1; 
+  let listItem = doc.createElement('li');
+  let guestNumber = guestsAdded; 
+  listItem.id = 'guest-' + guestNumber; 
+
+  let newTextInput = doc.createElement('input'); 
+  newTextInput.type = 'text';
+  newTextInput.name = 'guest-' + guestNumber
+
+  let newInputLabel = doc.createElement('label');
+  newInputLabel.htmlFor = 'guest-' + guestNumber;
+  newInputLabel.innerText = 'Guest: ';
+
+  // delete button associated with this guest
+  let removeGuestButton = doc.createElement('button');
+  removeGuestButton.innerHTML = 'X';
+  removeGuestButton.addEventListener('click', function() {removeGuest(listItem.id, doc);});
```

Comment:
Have you heard of arrow function experssions? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions
It's a more compact way of writing this.

Reply:
Would it be better to use the arrow function expression to call my removeGuest() function? 
i.e. removeGuestButton.addEventListener('click', ()=>(removeGuest(listItem.id, doc)));

Or should I use the arrow function to completely replace the removeGuest() function
i.e. removeGuestButton.addEventListener('click', ()=>(doc.getElementById(listItem.id).remove()));

Reply:
Either is probably fine although the former is probably easier to read.

Reply:
Went with the former